source/gui/addonGui.py

```diff
@@ -465,7 +465,7 @@ def installAddon(parentWindow, addonPath):
 
 	prevAddon = None
 	for addon in addonHandler.getAvailableAddons():
-		if not addon.isPendingRemove and bundle.name==addon.manifest['name']:
+		if not addon.isPendingRemove and bundle.name.lower()==addon.manifest['name'].lower():

```

Comment:
Did you look for other locations that compare name? I expected there to be more.

Reply:
Good point there. Actually, there are two things we should take care of:

* The name of the addon
* The name of the folder in which the addon resides

The add-on template makes sure that these are equal. NVDA seems to treat them separately, though.

Reply:
>> Actually, there are two things we should take care of

Is this PR ready to be merged, or does it need more work?